BackofficeBundle/Tests/Form/DataTransformer/ChoicesOptionToArrayTransformerTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Tests\Form\DataTransformer;
+
+use OpenOrchestra\BackofficeBundle\Form\DataTransformer\ChoicesOptionToArrayTransformer;
+use OpenOrchestra\BackofficeBundle\Form\DataTransformer\SuppressSpecialCharacterTransformer;
+
+/**
+ * Class ChoicesOptionToArrayTransformerTest
+ */
+class ChoicesOptionToArrayTransformerTest extends \PHPUnit_Framework_TestCase
+{
+    protected $suppressSpecialCharacter;
+    protected $transformer;
+
+    /**
+     * Set up the test
+     */
+    public function setUp()
+    {
+        $this->suppressSpecialCharacter = new SuppressSpecialCharacterTransformer();
```

Comment:
your test is not isolated, you should Phake the `SuppressSpecialCharacterTransformer` and write a test (if not already done) for it beside